msgpack-jackson/src/main/java/org/msgpack/jackson/dataformat/MessagePackGenerator.java

```diff
@@ -109,14 +108,7 @@ public MessagePackGenerator(int features, ObjectCodec codec, OutputStream out)
         }
         messageBufferOutputHolder.set(messageBufferOutput);
 
-        if (messagePacker == null) {
-            messagePacker = MessagePack.newDefaultPacker(messageBufferOutput);
-        }
-        else {
-            messagePacker.reset(messageBufferOutput);
```

Comment:
@marenzo The use of `reset()` here is for the performance. If we create a new MessagePacker instance every time, it will slow down the deserialization process.